roscala/src/test/scala/coop/rchain/rosette/PrimSpec.scala

```diff
@@ -0,0 +1,75 @@
+package coop.rchain.rosette
+
+import coop.rchain.rosette.prim.Number._
+
+import org.scalatest._
+
+class PrimSpec extends FlatSpec with Matchers {
+  val ctxt = Ctxt(
+    tag = null,
+    nargs = 1,
+    outstanding = 0,
+    pc = PC.PLACEHOLDER,
+    rslt = null,
+    trgt = null,
+    argvec = Tuple(1, Fixnum(1)),
+    env = null,
+    code = null,
+    ctxt = null,
+    self2 = null,
+    selfEnv = null,
+    rcvr = null,
+    monitor = null,
+    _slot = null
+  )
+
+  "fxPlus" should "correctly add numbers" in {
+    val newCtxt = ctxt.copy(nargs = 5, argvec = Tuple(5, Fixnum(1)))
+    fxPlus.fn(newCtxt) should be(Right(Fixnum(5)))
+  }
+
+  it should "fail for non-fixnum arguments" in {
+    val newCtxt = ctxt.copy(nargs = 5, argvec = Tuple(5, Ob.NIV))
+    fxPlus.fn(newCtxt) should be('left)
```

Comment:
I haven't seen this usage before.  How does the symbol `'left` end up meaning `Left(_)`?  Is it a feature of scalatest?